build_modules/test/summary_builder_test.dart

```diff
@@ -18,6 +18,10 @@ main() {
     setUp(() async {
       assets = {
         'build_modules|lib/src/analysis_options.default.yaml': '',
+        'a|lib/.meta_module':
+            '{"m":[{"p":["a","lib/a.dart"],"s":[["a","lib/a.dart"]],"d":[]}]}',
```

Comment:
nit: Create a MetaModule programatically and serialize it so this test is less brittle and easier to read

Reply:
Done.